### ISSUE TITLE
Time zone selector is wrong when current time zone is not part of the list

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#275]
> When I go to `https://simpleanalytics.com/example.com/settings` on desktop (e.g. Firefox) I see the following:
> 
> [ ] UTC
> [X] Europe/Berlin
> [ ] Other time zone (collapsed)
> 
> However, when I go to that page on mobile (e.g. Firefox), I see the following:
> 
> [ ] UTC
> [ ] CET
> [X] Other time zone (expanded) - GMT -12:00

What browser are you using on mobile vs desktop?

> On desktop I used Firefox (for Linux), where it was correct, and on mobile I used Firefox for Android, where it was wrong.

**Comment by adriaanvanrossum** [comment]
If you could check this one @hellodeibu

**Comment by adriaanvanrossum** [comment]
Customer:

> If you're using `Intl.DateTimeFormat().resolvedOptions().timeZone` in JavaScript to detect the time zone name, the above expression returns "Europe/Berlin" for me on Firefox for Linux but "CET" on Firefox for Android.
> 
> This looks like a bug in Firefox for Android, for which there's already an issue in Mozilla's bug tracker: https://bugzilla.mozilla.org/show_bug.cgi?id=1609714
> Hopefully this gets fixed soon.

**Comment by hellodeibu** [comment]
@adriaanvanrossum given that the bug is in Firefox (and on Android only, seemingly), known and their PR will eventually be merged into their main release channel, do you think it'll still be good to look for a way to see if we can work around this issue on our end too? As it's such a specific use-case that most customers won't run into, and even if they do is a mild inconvenience as no functionality actually breaks. If you like I could see if there's something we can do on our end of course, I just wanted to double check this before I do.

**Comment by adriaanvanrossum** [comment]
I agree. Let's focus on other issues for now and but let's keep this open to check back in 2 months if Firefox did fix it.

**Comment by srl295** [comment]
> They updated their issue to make it a duplicate of https://bugzilla.mozilla.org/show_bug.cgi?id=1534160. We can check the progress of this PR, it seems done, but not reviewed yet.
> 
> [unicode-org/icu#605](https://github.com/unicode-org/icu/pull/605). There is some movement in that PR in the recent days. Let's wait one more month.

@adriaanvanrossum merged.

**Comment by adriaanvanrossum** [comment]
Thanks for letting us know @srl295, appreciate it.
